ProjectTemplates/VisualStudio2010/DesktopGL/MonoGameDesktopGL.vstemplate

```diff
@@ -1,7 +1,7 @@
 <VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
   <TemplateData>
-    <Name>MonoGame Linux Project</Name>
-    <Description>A MonoGame game project for the Linux desktop.</Description>
+    <Name>MonoGame OpenGL Project</Name>
+    <Description>A MonoGame game project for Windows, Mac and Linux using OpenGL.</Description>
```

Comment:
I think we should expand this information a bit more.  We should remind people that this template generates one set of binaries that work across all three platforms.

Also I don't really like having OpenGL in the project name "MonoGame OpenGL Project".  Seems wrong to call out one API like that.  The intent here is to convey to the developer that this one project works across Windows, Mac, and Linux desktops.  This is why we called it the "DesktopGL" platform.  I'm thinking instead we do like...

 - "MonoGame Desktop Project"
 - "MonoGame Universal Desktop Project"
 - "MonoGame Cross Platform Desktop Project"
 - "MonoGame Win/Mac/Linux Desktop Project"

Thoughts?  @KonajuGames  @dellis1972  ??